TOOLS/ce/src/main/scala/org.opalj.ce/CommentParserWrapper.scala

```diff
@@ -0,0 +1,361 @@
+                        line = line.stripPrefix(",").trim
+                    } else if (line.startsWith("[")) {
+                        // Case: The following symbol opens a list
+                        value += parseList(nextComment)
+                        line = line.stripPrefix(",").trim
+                    } else if (
+                        line.startsWith("]") || (line.startsWith(",") && line.stripPrefix(",").trim.startsWith("]"))
+                    ) {
+                        // Case: The following symbol closes the list
+                        line = line.stripPrefix(",").trim.stripPrefix("]").trim
+                        break()
+                    } else if (line != "") {
+                        // Case: The following symbol is an entry
+                        value += parseEntry(nextComment)
+                        line = line.stripPrefix(",").trim
+                    }
+
+                    if (line == "" && iterator.hasNext) {
+                        // Load next line when done
+                        line = iterator.next().trim
+                    }
+                }
+            }
+            currentComment += getSingleLineComment
+
+            // Finish
+            ConfigList(value, DocumentationComment.fromString(currentComment))
+        }
+
+        /**
+         * Gets all comments until the next non-comment entry and writes them into a new ListBuffer.
+         * @return Returns a new ListBuffer with the content of the comment.
+         */
+        private def parseComments(): ListBuffer[String] = {
+            val comment = new ListBuffer[String]
+            parseComments(comment)
+        }
+
+        /**
+         * Gets all comments all comments until the next non-comment entry and writes them into an existing ListBuffer.
+         * @param comment Accepts the ListBuffer that the following comment should be added to.
+         * @return Returns the existing ListBuffer, but with the content of the comment added to it.
+         */
+        private def parseComments(comment: ListBuffer[String]): ListBuffer[String] = {
+            while (line.startsWith("#") || line.startsWith("//") || line == "") {
+                if (line != "") comment += getSingleLineComment
+                line = iterator.next().trim
+            }
+            comment
+        }
+
+        /**
+         * Adds a single line of Comment to the raw Comment string if the line has the comment flags
+         * @return returns an empty string if the next line is not a Comment. Returns the Comment without the comment flags if the line is a comment.
+         */
+        private def getSingleLineComment: String = {
+            if (line.startsWith("#") || line.startsWith("//")) {
+                // Add the comment in the same line of the list as well
+                val currentComment = line.stripPrefix("#").stripPrefix("//").trim
+                line = ""
+                return currentComment
```

Comment:
Remove the return and just add an `else` before the empty string to be returned in the other case

Reply:
Done